src/services/users/user_functions.ts

```diff
@@ -0,0 +1,23 @@
+import {MSUser} from '../../model/MSUser';
+import {MSContact} from '../../model/MSContact';
+import {BookitUser} from '../../model/BookitUser';
+
```

Comment:
No, the functions (mapMSUserToBookitUser, mapMSContactToBookitUser) that map from one type to another.  Sorry it wasn't clear.  Didn't know where else to drop the comment other than directly over the functions.  I looked for tests, but didn't see any.  Maybe I missed them.